tools/review_tracker.py

```
@@ -0,0 +1,389 @@
+                cursor.execute(
+                    "INSERT INTO reviews (path, last_reviewed) VALUES (?, NULL)",
+                    (file_path,)
+                )
+                added_count += 1
+            except sqlite3.IntegrityError:
+                # File already exists in database
+                existing_count += 1
+
+        self.conn.commit()
+
+        print(f"✓ Scan complete:")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)